utilities/run-test-suites/gsheetsImport/resultsImporter.py

```diff
@@ -25,35 +25,13 @@
 SCOPES = ['https://spreadsheets.google.com/feeds',
           'https://www.googleapis.com/auth/drive']
 
-# TEST_STATES = ["Pass", "Warning", "Fail", "Manual", "Not Applicable", "Not Implemented", "Test Disabled",
-#                "Could Not Test"]
-# Reorder the values to group by severity
+
```

Comment:
Just a suggestion that we leave a comment explaining why they're listed in this order:

```suggestion

# Test states grouped by severity
```